File: mRemoteNG/Connection/Connection.Protocol.Serial.vb
Namespace Connection
    Namespace Protocol
        Public Class Serial
            Inherits Connection.Protocol.PuttyBase

            Public Sub New()
                Me.PuttyProtocol = Putty_Protocol.serial
            End Sub

            Public Enum Defaults
                Port = 9600
            End Enum
        End Class
    End Namespace
End Namespace

File: mRemoteV1/Connection/Protocol/deprecated/Connection.Protocol.Serial.cs


namespace mRemoteNG.Connection.Protocol
{
	public class Serial : PuttyBase
	{
				
		public Serial()
		{
			this.PuttyProtocol = Putty_Protocol.serial;
		}
				
		public enum Defaults
		{
			Port = 9600
		}
	}
}
